Source/GameBaseFramework/Public/BlueprintLibraries/GBFMathBlueprintLibrary.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include <CoreMinimal.h>
+#include <Kismet/BlueprintFunctionLibrary.h>
+
+#include "GBFMathBlueprintLibrary.generated.h"
+
+UCLASS()
+class GAMEBASEFRAMEWORK_API UGBFMathBlueprintLibrary : public UBlueprintFunctionLibrary
+{
+    GENERATED_BODY()
+
+public:
+    static int GetWrappedIndex( const int index, const int size )

```

Comment:
UFUNCTION( BlueprintPure )

if not possible, UFUNCTION( BlueprintCallable )

Reply:
Done, BlueprintPure now.